sphinx_needs/directives/needimport.py

```diff
@@ -92,9 +92,20 @@ def run(self) -> Sequence[nodes.Node]:
             )[1]
 
             if not os.path.exists(correct_need_import_path):
-                raise ReferenceError(
+                warning_text = (
                     f"Could not load needs import file {correct_need_import_path}"
                 )
+                log_warning(
+                    logger,
+                    warning_text,
+                    "needimport",
```

Comment:
A different subtype would make sense for this case, so it can be ignored isolated.

```suggestion
                    "needimport_missing",
```